tests/ert3/data/test_transmitters.py

```diff
@@ -0,0 +1,141 @@
+import contextlib
+import json
+import os
+import pathlib
+import pickle
+import tempfile
+from typing import Callable, ContextManager
+
+import cloudpickle
+import pytest
+from ert3.data import (
+    InMemoryRecordTransmitter,
+    SharedDiskRecordTransmitter,
+    RecordTransmitter,
+)
+from tests.utils import tmpdir
+
+
+@contextlib.contextmanager
+def prefect_factory_context():
+    tmp_path = tempfile.TemporaryDirectory()
+    tmp_storage_path = pathlib.Path(tmp_path.name) / ".shared-storage"
+    tmp_storage_path.mkdir(parents=True, exist_ok=True)
```

Comment:
Is the `exist_ok` necessary?

Reply:
No, leftover from earlier implementation